### DISCUSSION TITLE
How do I include a related field in a pydantic model?

### DISCUSSION THREAD

**Discussion by noahnu** [tortoise/tortoise-orm#939] [Q&A]
```
class Comment(Model):
    id = fields.UUIDField(pk=True)
    author = fields.ForeignKeyField("models.User", related_name="comments")

Comment_Pydantic = pydantic_model_creator(Comment, name="Comment", include=["id", "author_id"])
```

where the author model has a "display_name" field, how can I include display_name in Comment_Pydantic? When I try `include=["author__display_name"]`, no field is added. When I try `include=["author"]`, I get errors saying "response -> 0 -> author -> [some field on author]" / "field required (type=value_error.missing)". I imagine the issue in the latter case is then I have to fetch all of author whereas I'm just prefetching "author__display_name". Is there a way to only include one field on the relation?

**Comment by bekha-io** [comment]
Are there any updates on it?

**Comment by plusiv** [comment]
Any updates?

**Comment by vlakius** [comment]
Had this problem, Try to look at Early init 
https://tortoise.github.io/examples/pydantic.html?h=early+init
